codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/codegen/SymbolVisitor.java

```diff
@@ -574,29 +578,15 @@ public Symbol structureShape(StructureShape shape) {
       var referredShape = model.expectShape(
         shape.expectTrait(ReferenceTrait.class).getReferentId()
       );
-      var isService = shape.expectTrait(ReferenceTrait.class).isService();
-      if (isService) {
-        builder.putProperty(
-          "Referred",
-          symbolBuilderFor(
-            referredShape,
-            "Client",
-            SmithyNameResolver.shapeNamespace(referredShape)
-          )
-            .putProperty(SymbolUtils.POINTABLE, true)
-            .build()
-        );
-      } else {
-        builder.putProperty(
-          "Referred",
-          symbolBuilderFor(
-            referredShape,
-            "I".concat(getDefaultShapeName(referredShape))
-          )
-            .putProperty(SymbolUtils.POINTABLE, false)
-            .build()
-        );
-      }
+      builder.putProperty(
+        "Referred",
+        symbolBuilderFor(
+          referredShape,
+          "I".concat(getDefaultShapeName(referredShape))
```

Comment:
Does dependencies pass with this change?

Reply:
Made a change so that dependencies passes. Also merged with Golang/dev so that dependencies could run in CI.